README.md

```diff
@@ -0,0 +1,53 @@
+# Welcome to cascade 👋
+[![Documentation](https://img.shields.io/badge/documentation-yes-brightgreen.svg)](https://github.com/the-dr-lazy/cascade/wiki)
+[![Twitter: the\_dr\_lazy\_](https://img.shields.io/twitter/follow/the\_dr\_lazy\_.svg?style=social)](https://twitter.com/the\_dr\_lazy\_)
+https://github.com/the-dr-lazy/cascade/blob/master/LICENSE
+> A project management service just to show that real-world Haskell is awesome.
+
+### 🏠 [Homepage](https://github.com/the-dr-lazy/cascade)
+
+### ✨ [Demo](www.demo.com)
+
+## Install
+
+```sh
+install command
```

Comment:
So, the plan is to distribute binary for Linux and OS X (and maybe Windows). Until then we just should state the project is under heavy development and the user just can install it via building it from scratch.

Reply:
I do not have enough information about the installation, I just left this section for you to complete